main.go

```diff
@@ -220,7 +222,9 @@ func main() {
 		w := tabwriter.NewWriter(os.Stdout, 1, 1, 1, ' ', 0)
 		fmt.Fprintln(w, "Masked Email\tFor Domain\tState\tLast Email At\t")
 		for _, email := range maskedEmails {
-			fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", email.Email, email.ForDomain, email.State, email.LastMessageAt)
```

Comment:
I've realized that my fix for not listing "deleted" emails is part of that PR as well.

Reply:
Shall I remove it? Or could we add a flag for "--show-deleted" ?